calculator/android/app/build.gradle

```diff
@@ -0,0 +1,67 @@
+plugins {
+    id "com.android.application"
+    id "kotlin-android"
+    id "dev.flutter.flutter-gradle-plugin"
+}
+
+def localProperties = new Properties()
+def localPropertiesFile = rootProject.file('local.properties')
+if (localPropertiesFile.exists()) {
+    localPropertiesFile.withReader('UTF-8') { reader ->
+        localProperties.load(reader)
+    }
+}
+
+def flutterVersionCode = localProperties.getProperty('flutter.versionCode')
+if (flutterVersionCode == null) {
+    flutterVersionCode = '1'
+}
+
+def flutterVersionName = localProperties.getProperty('flutter.versionName')
+if (flutterVersionName == null) {
+    flutterVersionName = '1.0'
+}
+
+android {
+    namespace "com.example.calculator"
```

Comment:
Please follow the readme properly.

<img width="1068" alt="Screenshot 2023-11-03 at 09 19 36" src="https://github.com/nisrulz/flutter-examples/assets/2096087/e4e85e9e-2f2c-4812-a2f0-4e22c4f711c9">